.pipelines/e2e-job-template.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     CLEANUP_ON_EXIT: true
     CLEANUP_IF_FAIL: true
     RETAIN_SSH: false
-    ENABLE_KMS_ENCRYPTION: true
+    ENABLE_KMS_ENCRYPTION: false
```

Comment:
why change the e2e default here?

Reply:
Temporary. I have yet to implement the no_template for KMS.

Reply:
still needed?

Reply:
I still haven't implemented it. I will have to [WIP] my PR.